warehouse/ingest-core/src/main/java/datawave/ingest/data/config/ingest/AbstractIngestHelper.java

```diff
@@ -64,7 +65,9 @@ public String getNormalizedMaskedValue(final String key) {
                 final Set<String> normalizedValues = normalizeFieldValue(fieldName.toUpperCase(), value);
                 return normalizedValues.iterator().next();
             } catch (final Exception ex) {
-                log.warn(this.getType().typeName() + ": Unable to normalize masked value of '" + value + "' for " + fieldName, ex);
+                if (log.isWarnEnabled()) {
```

Comment:
would it make sense to output this regardless of if the warn level is enabled or not?